### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LukasHirt**
## Open tasks:
- [ ] Add entry for OIDC

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4886/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/4886/

<p>

















</p></details>

File: README.md [hunk 1]
```diff
@@ -2,27 +2,42 @@
...
+```
+
+## Create config.json
+- For reference look into config.json.sample (example with oauth2)
+
 ## Run
 
 - optionally provide custom domain name: `export SERVER_HOST=0.0.0.0:8300`
-- run a webpack dev server `yarn run watch`
+- run a webpack dev server `yarn run watch` (`yarn run watch-all` if you want to watch apps as well)
```
Comment (felixheidecke) [inline README.md]: As we are omitting the `run` command in Line 12 we might as well be consistent and drop it here as well ;-)

Comment (LukasHirt) [reply README.md]: Makes sense. Will do

**Pull request merged**